pandas/io/formats/latex.py

```diff
@@ -73,10 +73,8 @@ def __init__(
         self.multicolumn_format = multicolumn_format
         self.multirow = multirow
         self.clinebuf: List[List[int]] = []
-        self.strcols = self._get_strcols()
-        self.strrows: List[List[str]] = list(
-            zip(*self.strcols)  # type: ignore[arg-type]
-        )
+        self.strcols: Sequence[Sequence[str]] = self._get_strcols()
+        self.strrows: Sequence[Sequence[str]] = list(zip(*self.strcols))
```

Comment:
The variable type annotations can be removed. My preference is not to add variable types annotations unless absolutely necessary.

the variable types for self.strcols and self.strrows can (should) be inferred from the rhs expression.

self.strcols:  Revealed type is 'builtins.list[builtins.list[builtins.str]]'
self.strrows: Revealed type is 'builtins.list[builtins.tuple*[Any]]'

I'm not sure why mypy can't infer more precise for self.strrows

mypy is green, but if wanted to be more precise for self.strrows could use `List[Tuple[str, ...]]`

Reply:
Thank you for the suggestions!
Updated accordingly.